lua/modules/linear_bolzmann_solvers/do_transient/lbts_lua_create_solver.cc

```diff
@@ -5,7 +5,7 @@
 #include "framework/runtime.h"
 
 #if 0
-namespace lbs::lbts_lua_utils
+namespace lbts_lua_utils
```

Comment:
We should consider renaming these things like `lbts` at some point since they no longer make a whole lot of sense with the solver being `DiscOrdTransientSolver`. Future work, not here.

Reply:
Also added into #57 as a task